core/src/main/java/com/datastrato/gravitino/connector/CatalogOperations.java

```diff
@@ -18,6 +18,9 @@
 @Evolving
 public interface CatalogOperations extends Closeable, HasPropertyMetadata {
 
+  String MYSQL_CATALOG_CONNECTION_CLEAN_UP_THREAD =
+      "com.mysql.cj.jdbc.AbandonedConnectionCleanupThread";
```

Comment:
Why do we define the string here?

Reply:
I want to make it accessible for modules `Iceberg` and `JDBC-MySQL`, so I put it here.  Is it better to move them to module `common`?

Reply:
Do you need to expose this user?

Reply:
> Do you need to expose this user?

No,  it is just for inner use. `common` will be directly exposed to users, so common is also not suitable for it.

Reply:
If it should not be exposed to user, why don't you put it here?

Reply:
My original intention was to make `MYSQL_CATALOG_CONNECTION_CLEAN_UP_THREAD` sharable. Since different catalogs only share the `api`, `common`, `core`, `server` and `server-common`  jars.
- `api`, `common` are directly exposed to client, it's not suitable to put it here
- `server` and `server-common` are only for server-specified contents, it's also not suitable
-  The only suitable place it put it in the `core` package. 

Then, we have the following solution.
- Move `MYSQL_CATALOG_CONNECTION_CLEAN_UP_THREAD` to a class that holds constant values.
- Do not make `MYSQL_CATALOG_CONNECTION_CLEAN_UP_THREAD` sharable, just use it as a literal constant in each catalog. 

Since we do not have a class for contact values, I will use the latter one to address these comments.